core/templates/dev/head/editor/ExplorationPreview.js

```diff
@@ -59,5 +61,31 @@ oppia.controller('ExplorationPreview', [
     $scope.$on('playerStateChange', function() {
       $scope.allParams = LearnerParamsService.getAllParams();
     });
+
+    explorationData.getData().then($scope._refreshPreviewTab = function() {
```

Comment:
Two things:

(1) Let's drop the underscore -- we're moving away from that convention, and we're going to use "var" instead of "$scope." for private-to-JS methods. So, "var refreshPreviewTab" instead of "$scope._refreshPreviewTab".

(2) I would suggest declaring refreshPreviewTab as a top-level function in this controller, then doing:

  explorationData.getData().then(refreshPreviewTab);